src/cgame/rocket/rocket_documents.cpp

```diff
@@ -168,3 +169,30 @@ void Rocket_DocumentAction( const char *name, const char *action )
 		}
 	}
 }
+
+static bool debugger_initialised = false;
+void CG_RmlUIDebuggerToogle()
+{
+	if (!debugger_initialised)
+	{
+		Rml::Debugger::Initialise(*cgs.mapname // CHECKME
```

Comment:
You probably want an argument for hud/menu or at least allow for an override because people may want to debug ingame menus and such when a map is loaded. Furthermore, iirc, we never inject mouse/input events into the HUD context, so you can't actually use the debugger in that context (I think...)

Reply:
Ah, wait, are the in-game menu part of the menu or HUD context?

Reply:
Anything clickable is always the menuContext (ie, if you're in game and press ESC, you are in the menuContext). The hudContext is exclusively the the HUD (ie, things you can't click on because your mouse/keyboard are captured by the game. The only thing you you might actually want to click on is the tab scoreboard, which is part of the hud context.

Reply:
Then it makes things more complicated for the keybindings in particular. I'm not sure what would be the correct, non-surprising behavior. Use two keys? Press the same key several times (none->menu->hud->none)?

Reply:
In games like CSGO, which have a similar scoreboard, if you open a clickable HUD element, you right click to bring up a mouse.

Clickable HUD elements might be interesting if we ever want to be cool like Doom3 and have clickable menus in the map.